### PR TITLE
Feature/176092178 add hdf5 project file to download

### PR DESCRIPTION
**Pull request by lingyun1010**
This PR comes with https://github.com/ebi-gene-expression-group/atlas-web-core/pull/105

### CODE REVIEW

File: app/src/main/java/uk/ac/ebi/atlas/download/ExperimentFileLocationService.java [hunk 1]
```diff
@@ -30,6 +30,8 @@ public ExperimentFileLocationService(DataFileHub dataFileHub) {
     @Nullable
     public Path getFilePath(String experimentAccession, ExperimentFileType fileType) {
         switch (fileType) {
+            case HDF5:
+                return dataFileHub.getSingleCellExperimentFiles(experimentAccession).projectHdf5.getPath();
```
Comment (upendrakumbham) [inline app/src/main/java/uk/ac/ebi/atlas/download/ExperimentFileLocationService.java:34]: Can you think of any test case to test this new file type?

**Review by upendrakumbham — CHANGES_REQUESTED** [review]
The code looks fine, please cover tests for this new feature.

File: app/src/main/java/uk/ac/ebi/atlas/download/ExperimentFileLocationService.java (hunk 1 above)
Comment (lingyun1010) [reply app/src/main/java/uk/ac/ebi/atlas/download/ExperimentFileLocationService.java:34]: We don't really have HDF5 file in the existing test experiments.

**Review by upendrakumbham — APPROVED** [review]
LGTM!

**Pull request merged**